testomania/src/main/java/com/earth/testomania/technical/presentation/QuizViewModel.kt

```diff
@@ -35,7 +35,7 @@ class QuizViewModel @AssistedInject constructor(
     val loading = _loading.asStateFlow()
 
     private val _error = MutableSharedFlow<Int>()
-    val error = _loading.asStateFlow()
+    val error = _error
```

Comment:
I think you forgot about `_error.asSharedFlow()`

Reply:
You are right, I fix it.